### PR TITLE
feat(build): Check for minimum nix version before running nixci

### PR DESCRIPTION
**Pull request by shivaraj-bh**
resolves https://github.com/srid/nixci/issues/62

### CODE REVIEW

File: src/checks.rs
Comment (srid) [inline src/checks.rs:1]: This looks like a repeat of code from nix-health; anyway we can (expose and) use upstream API directly?

Comment (shivaraj-bh) [reply src/checks.rs:1]: Good idea, will do that tomorrow.

Comment (shivaraj-bh) [reply src/checks.rs:1]: https://github.com/juspay/nix-health/pull/25

**Review by srid — APPROVED** [review]

**Comment by srid** [comment]
Huh, I shouldn't have merged this. I thought the CI passed, but I mixed this up with the nix-health PR, and merged it. #77

**Pull request merged**